cli/cli.go

```diff
@@ -61,13 +71,22 @@ type flagopts struct {
 	OutputCompact bool              `short:"c" long:"compact-output" description:"output without pretty-printing"`
 	OutputIndent  *int              `long:"indent" description:"number of spaces for indentation"`
 	OutputTab     bool              `long:"tab" description:"use tabs for indentation"`
-	OutputYAML    bool              `long:"yaml-output" description:"output in YAML format"`
+	OutputYAML    bool              `short:"y" long:"yaml-output" description:"output in YAML format"`
+	OutputXML     bool              `short:"x" long:"xml-output" description:"output in XML format"`
```

Comment:
(Note: I'm not the author of gojq, just an interested third party.)

I think it makes sense to add even more formats in the future (toml, msgpack, bson, etc), but this list of boolean flags will grow unmanageable.

What do you think about `--input-format=xml` and `--output-format=xml` instead of these format-specific parameters?

Reply:
Sure, looks to be a very good idea :)